pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ServiceChannelInitializer.java

```diff
@@ -64,7 +65,7 @@ public ServiceChannelInitializer(ProxyService proxyService, ProxyConfiguration s
         if (enableTls) {
             if (tlsEnabledWithKeyStore) {
                 serverSSLContextAutoRefreshBuilder = new NettySSLContextAutoRefreshBuilder(
-                        serviceConfig.getTlsProvider(),
+                        serviceConfig.getServiceSslProvider(),
```

Comment:
Why not to use `tlsProvider`?

Reply:
And the webservice, websocket also use the `tlsProvider` as `sslProviderString` for `JettySslContextFactoryWithAutoRefresh`

Reply:
We have to know the SSLContext provider and SSL provider:

- The SSLContext provider is used to new SSLContext
-  The SSL provider is used to handle the SSL(This is my understand). 

The `tlsProvider` is the SSLContext provider, not the SSL provider. 

Broker service:

With KeyStore, we only need to set the SSLContext provider, the current implementation doesn't support setting the SSL provider.

With CACert, we only need to set the SSL provider, the default should be OpenSSL, when the OpenSSL is not available, will use JDK.

Web service:

SSL context provider and SSL provider are the same.


When both use the KeyStore, we can use `tlsProvider`. When use the CACERT, the web service and broker service cannot use the same provider, so we need to split these config.

Reply:
There looks like I need to compatible the `tlsProvider` when using KeyStore?